client/src/utils/getEnv.ts

```diff
@@ -0,0 +1,15 @@
+export const getEnvSykmeldingerRoot = () => {
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
Dette gjelder forsåvidt for q1 også.

Jeg har en `.env` fil i `/client/src/` som står beskrevet i readme. Synes enten vi skal bruke det eller sette hver environment variabel i startscriptet i `package.json`: 
`"start": "REACT_APP_SYKMELDINGER_ROOT=/ REACT_APP_SYKEFRAVAER_ROOT=/ craco start`

Ved å hente prosessvariable i koden er det lett ved første øyekast å se at den kommer fra utenfor applikasjonen. En wrapper rundt dette vil være én ekstra ting å vedlikeholde

Reply:
Enig, var usikker på hva som var best practice her og hadde ikke sett den readme-en. Da gjør vi det på den måten 👍